server/common/transport-shared/src/main/java/org/kaaproject/kaa/server/transport/session/SessionAware.java

```diff
@@ -13,10 +13,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.kaaproject.kaa.server.operations.service.akka.messages.io.request;
-
-import org.kaaproject.kaa.server.operations.service.netty.NettySessionInfo;
+package org.kaaproject.kaa.server.transport.session;
 
+/**
+ * Marker interface indicating that a class has a basic knowledge about related
```

Comment:
A marker interface indicating that a class has a basic knowledge about the related session.